packages/storybook-a11y-test/stories/Button.stories.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import {ComponentStory, ComponentMeta} from '@storybook/react';
+
+import {Button} from './Button';
+
+// More on default export: https://storybook.js.org/docs/react/writing-stories/introduction#default-export
+export default {
+  title: 'Example/Button',
+  component: Button,
+  // More on argTypes: https://storybook.js.org/docs/react/api/argtypes
+  argTypes: {
+    backgroundColor: {control: 'color'},
+  },
+} as ComponentMeta<typeof Button>;
+
+// More on component templates: https://storybook.js.org/docs/react/writing-stories/introduction#using-args
+const Template: ComponentStory<typeof Button> = (args) => <Button {...args} />;
+
+export const Primary = Template.bind({});
```

Comment:
If we're all about modern behaviours, then we could try out [CSF v3's new object format](https://storybook.js.org/blog/component-story-format-3-0/#new-in-csf-3-0)

Something like: 
```js
export const Primary = {args: { primary: true, label: 'Button' }}
```